.azure-pipelines-templates/matrix.yml

```diff
@@ -53,7 +53,7 @@ parameters:
     san:
       cmake_args: '-DSAN=ON'
     perf:
-      cmake_args: '-DPERF=ON'
+      cmake_args: '-DISTRIBUTE_PERF_TESTS=`../.nodes.sh`'
```

Comment:
Not sure where to write this: now that we've got distributed machines, the forwarding test with signatures can be run with a client for each backup  (`-cn` option).

Reply:
Please open an issue.